### ISSUE TITLE
Wikidata PageRank scores

### ISSUE THREAD

**Issue by athalhammer** [europeana/entity-api-import#32]
Hi everyone,

I just wanted to let you know that I find it great that the Wikidata PageRank scores are of use for you!

If you would like to use a more recent version, please check out https://github.com/athalhammer/danker (there is a download section). Also, I am curious if, or probably I should ask, **why** you found the version that doesn't factor in links from the English language Wikipedia more useful (https://web.archive.org/web/20180222182923/https://people.aifb.kit.edu/ath/).

Keep up the great work!

**Comment by gsergiu** [comment]
Hi @athalhammer and thank you for your input.
It is great to hear that a new version is available. 
We started with an experiment that didn't use the factor, and somehow we stick to it.
It would be probably good to summarize the difference near the names of the files:
    wd_pr_ultimate.ttl.bz2 (Turtle, 82M)
    wd_pr_ultimate.tsv.bz2 (TSV, 131M)
    wd_pr_en.ttl.bz2 (Turtle, 33M)
    wd_pr_en.tsv.bz2 (TSV, 54M)
    wd_pr_ultimate-minus-en.ttl.bz2 (Turtle, 68M)
    wd_pr_ultimate-minus-en.tsv.bz2 (TSV, 106M)

if you are interested in the way we use the PageRank, you might want to check this [paper](https://www.researchgate.net/publication/344427829_Named_Entity_Recommendations_to_Enhance_Multilingual_Retrieval_in_Europeanaeu).

**Comment by athalhammer** [comment]
> It would be probably good to summarize the difference near the names of the files:

Yeah, that is not relevant any longer. The files are old and offline and only the website is still on archive.org. However, if you try out the newer file produced by [danker](https://github.com/athalhammer/danker ) and you find that it has less good results get back to me and I can see what we can do. Back then it seemed like an interesting line of research to factor out the bias contributed by the biggest Wikipedia.

**Comment by gsergiu** [comment]
We were satisfied with the PageRank from the old files, but we are also further developing our systems going more in the direction of recommending entities available in a specific language. For this purpose, some normalization which is language related might be of help. We are taking in consideration to use the new values for PageRanks. It might be a good idea to have a short conversation when we start this work. 

Br,
Sergiu

**Comment by aisaac** [comment]
Hi @athalhammer thanks for reaching out! As @gsergiu said we have a part on our (not yet public) search strategy about "Autosuggest ranking criteria". With an action that reads 
> ACTION: We will reach out to the designers of Wikidata PageRank for this line of work (also to inform them on our use of their research).

So it's great you have found us!
This is rather planned for a later stage though: we have many, many things to do related to search. Maybe up to a couple of year, I'm afraid :-/
In the meantime, may I ask how you have found us?

**Comment by athalhammer** [comment]
@gsergiu @aisaac, thanks for the nice words and just get back to me if you need anything. Also, if, at some point, you could link to the new project website and/or respective research paper would be great.

Finding you was not that hard, I just used Google Scholar to search for "wikidata pagerank" and one of your deliverables popped up.
